code/game/objects/structures.dm

```diff
@@ -55,6 +55,7 @@
 
 /obj/structure/New()

```

Comment:
/obj/structure/New() was defined in two places. This is the consolidation of one